pages/features/cache.mdx

```diff
@@ -0,0 +1,110 @@
+# FlakeHub Cache
+
+*FlakeHub Cache* is a [Nix binary cache][cache] that works in conjunction with [FlakeHub][home].
+It has two main features that distinguish it from other Nix caching services:
+
+1. It enables you to [control access](#access-control) at the [flake][flakes] level.
```

Comment:
IIRC the access control is at the repository level. If you have one flake per repo that's the same thing, but you could be releasing multiple flakes from the same repo, and they would share the same cache by default.